dynamic_programming/subset_generation.py

```diff
@@ -1,44 +1,44 @@
-# Print all subset combinations of n element in given set of r element.
+# Return all subset combinations of n element in given set of r element.
 
 
 def combination_util(arr, n, r, index, data, i):
```

Comment:
Please add type hints to the function signature.

Reply:
i changed the code so do check it once